src/outlet/ticketStorage.ts

```diff
@@ -53,11 +55,22 @@ export type DecodedToken = DecodedTokenData & {
 }
 
 export interface DecodedTokenData {
-	devconId: string
+	tokenId?: string
+	eventId?: string
 	ticketIdNumber?: number
```

Comment:
Hi, @micwallace . As we do big updates then lets decide to use one field  ticketIdNumber(numeric, must be unique) or ticketIdString(string, hash it to make numeric value for NFT_ID) and name it ticketId
what do you think ? 
@nicktaras

Reply:
For EAS it is always a string type. This only has implications for smart contracts usage which is done at the application level, so I think this is a good idea. For old ASN attestations with ticketIdNumber we can just convert it into a string.

Reply:
Should we change ticketIdNumber to this below for correctness that this is provided in different types? Name and types.

export interface DecodedTokenData {
	tokenId?: string
	eventId?: string
	ticketId?: number | string"
}

Reply:
@nicktaras Yeah I suppose we can just let the user convert to string if it's needed.